backend/models/user.js

```diff
@@ -0,0 +1,42 @@
+import {Schema, Model} from 'mongoose';
+import {genSalt, hash} from 'bcrypt';
```

Comment:
do ES6 imports work in node? i don't believe so

Reply:
They do, but they're technically not recommended since the spec could change. I don't really think they're a problem for our scope.

Reply:
not a problem if we use typescript